test.md

```diff
@@ -0,0 +1,318 @@
+---
+description: Smoketest page
+title: Testing page
+---
+
+# Heading 1
+
+Plain block of text.
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+culpa qui officia deserunt mollit anim id est laborum.
+
+## Heading 2
+
+Text with various styles, basic markdown formatting.
+
+Once you create or link to a repository in Docker Cloud, you can set up [automated testing](/docker-cloud/builds/automated-testing.md) and [automated builds](/docker-cloud/builds/automated-build.md).
+
+![a small cute image](/images/footer_moby_icon.png)
+
+> **Note**: This is a note. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
+
+![a pretty wide image](/images/banner_image_24512.png)
+
+<center>
```

Comment:
We should probably be testing the correct usages instead of all of our various off-the-rails behavior. This can be both a demo and a unit test if we figure out how to properly implement common scenarios.